comp/logs/agent/agentimpl/serverless.go

```diff
@@ -7,6 +7,7 @@ package agentimpl
 
 import (
 	"context"
+	flareController "github.com/DataDog/datadog-agent/comp/logs/agent/flare"
```

Comment:
Do we have flares within the serverless context?

Reply:
I had to add it otherwise a null crash would happen here
https://github.com/DataDog/datadog-agent/commit/3af304da7c2cd4dd9de77ae59d5f8c8d9a29cee0